File: app/src/main/java/com/example/tourethiopia/FirstActivity.java
package com.example.tourethiopia;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class FirstActivity extends AppCompatActivity {
    int SPLASH_TIME = 4000; //This is 4 seconds

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_first);

        new Handler().postDelayed(new Runnable() {

// Using handler with postDelayed called runnable run method

            @Override

            public void run() {

                Intent i = new Intent(FirstActivity.this, MainActivity.class);
                startActivity(i);

                // close this activity

                finish();

            }

        }, SPLASH_TIME); // wait for 4 seconds


    }
}

File: app/src/main/java/com/example/tourethiopia/FirstActivity.kt
package com.example.tourethiopia

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.support.v7.app.AppCompatActivity

class FirstActivity : AppCompatActivity() {
    var SPLASH_TIME = 4000 //This is 4 seconds
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_first)
        Handler().postDelayed({
            val i = Intent(this@FirstActivity, MainActivity::class.java)
            startActivity(i)

            // close this activity
            finish()
        }, SPLASH_TIME.toLong()) // wait for 4 seconds
    }
}